src/main/java/ch/njol/skript/expressions/ExprWorldBorderSize.java

```diff
@@ -0,0 +1,77 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.WorldBorder;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Size of World Border")
+@Description("The size of a world border.")
+@Examples("set world border radius of {_worldborder} to 10")
+@Since("INSERT VERSION")
+public class ExprWorldBorderSize extends SimplePropertyExpression<WorldBorder, Double> {
+	static {
+		register(ExprWorldBorderSize.class, Double.class, "world[ ]border (size|diameter|:radius)", "worldborders");
+	}
+
+	private boolean radius;
+
+	@Override
+    public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+        radius = parseResult.hasTag("radius");
+        setExpr((Expression<? extends WorldBorder>) exprs[0]);
+        return true;
+    }
+
+	@Override
+	@Nullable
+	public Double convert(WorldBorder worldBorder) {
+		return worldBorder.getSize() * (radius ? 0.5 : 1);
+	}
+
+	@Override
+	@Nullable
+	public Class<?>[] acceptChange(ChangeMode mode) {
```

Comment:
```suggestion
	public Class<?> @Nullable [] acceptChange(ChangeMode mode) {
```
for all other classes too